src/scripts/website.py

```diff
@@ -47,8 +47,18 @@ def configure_build(botan_dir, build_dir):
                    '--with-doxygen', '--with-sphinx',
                    '--with-build-dir=%s' % (build_dir)])
 
-def run_doxygen(tmp_dir, output_dir):
-    run_and_check(['doxygen', os.path.join(tmp_dir, 'build/botan.doxy')])
+def run_doxygen(botan_dir, tmp_dir, output_dir):
+
+    doxygen_config = os.path.join(tmp_dir, 'build/botan.doxy')
+    with open(doxygen_config, 'r') as f:
+        config_content = f.read()
+    
+    config_content = config_content.replace('STRIP_FROM_PATH        =', f'STRIP_FROM_PATH        = {os.path.dirname(botan_dir)}')
```

Comment:
This may well be the fix that's required, but this the approach. This file is already created from a template (`src/build-data/botan.doxy.in`) so what we'd need to do is edit that template with something like

```
diff --git a/src/build-data/botan.doxy.in b/src/build-data/botan.doxy.in
index 60f661716..bc5e0dfa9 100644
--- a/src/build-data/botan.doxy.in
+++ b/src/build-data/botan.doxy.in
@@ -13,7 +13,7 @@ ABBREVIATE_BRIEF       =
 ALWAYS_DETAILED_SEC    = NO
 INLINE_INHERITED_MEMB  = YES
 FULL_PATH_NAMES        = YES
-STRIP_FROM_PATH        =
+STRIP_FROM_PATH        = %{abs_root_dir}
 STRIP_FROM_INC_PATH    =
 SHORT_NAMES            = NO
 JAVADOC_AUTOBRIEF      = NO
```

Here `abs_root_dir` is a value set by `configure.py`. You can see all of the values that feed into the templates in `build/build_config.json`. This may not be the right variable to use here I'd have to double check this.

Reply:
It looks like it will be cleaner and more understandable than what happened with configure.py, which I wrote later and as you reported. I will make the necessary arrangements and take a general look. If you tell me the points I need to test, I can check it.